### DISCUSSION TITLE
proxy subdomains in docker

### DISCUSSION THREAD

**Discussion by ghost** [coder/code-server#4563] [Q&A] [answered]
Hi,

Great project!. Apologies if this has been documented / answered before.

I am trying to set up subdomain proxy using the docker image codercom/code-server but can't seem to get it to work.
For example 5000.mydomain.com loads code server instead of the webapp.

in the usage docs it says to use this startup flag: --proxy-domain <domain>
However I can't find where to put it in docker?

It may also be relevant to say I am using jwilder/nginx-proxy image for this.

Thanks

**Comment by jsjoeio** [comment]
> For example 5000.mydomain.com loads code server instead of the webapp.

Can you elaborate here? I thought that's what you wanted? Also can you post your Dockerfile?

I wonder if you need to adjust your nginx config. I'm not super familiar but I'm sure we can help you

**Comment by ghost** [comment]
Hi thanks for the response. Code server is hosted on mydomain.com. I was reading in usage of this - https://coder.com/docs/code-server/latest/guide#proxying-to-create-a-react-app

So my intent was for 3000.mydomain.com in this case to load the development server of create-react-app (like /proxy/3000 does) instead it loads code server.

My Dockerfile? im using this image https://hub.docker.com/r/codercom/code-server

to build (roughly):

docker run -d \
   --name=coder \
   --restart=unless-stopped  \
   -e PGID=1000 -e PUID=1000  \
   -e password=password \
   -e VIRTUAL_HOST=mydomain.com,3000.mydomain.com,5000.mydomain.com \
   -e VIRTUAL_PORT=8080  \
   -e LETSENCRYPT_HOST=mydomain.com,3000.mydomain.com,5000.mydomain.com  \
   -e LETSENCRYPT_EMAIL=email  \
   -v "/home/user/workspaces:/home/coder/project"
   -v "/mnt:/mnt"
   -v "/opt:/host_opt"
   -v "/home/user:/home/coder/host"
  --restart=unless-stopped  \
codercom/code-server:latest

Sorry if this doesn't clear up the questions

**Comment by ghost** [comment, marked as answer]
Figured it out. Went into the config.yml and added proxy-domain: mydomain.com rather than using a flag and now works as expected.

**Reply by jsjoeio** [reply]
Ah-ha! Nice!
